src/styles/main.css

```diff
@@ -1,3 +1,7 @@
+html {
+  height: 100%;
+}
+
 /* Gradient doesnt work with styled-components */
```

Comment:
Not sure if this is doing anything?

Reply:
Note the white space at the bottom when I filter for nothing

![image](https://user-images.githubusercontent.com/44789534/163578312-19347c02-b739-4d9e-97ac-05ddc0a6b755.png)

Reply:
Tried for like 15 mins to fix this, no clue why this isnt working. Supposedly setting the html to 100% height and then body to min-height 100% is the way to go, which I did and it looked like the body encompassed the whole page. But then the root div even when set to min-height 100%, wouldnt fill. Made me mad so I just left as is lol, I'll add a TODO to look into later

Reply:
Bro I did the same thing and found myself spinning in circles. Agree with leaving it as a TODO for now